src/main/java/report_screens/ManipulateReviewDs.java

```diff
@@ -0,0 +1,97 @@
+package report_screens;
+
+
+import entities.Review;
+import java.io.*;
+import java.util.Scanner;
+
+
+//This class is used to update the database with the manipulated review object
+public class ManipulateReviewDs {
+
+    private static final String NAME_OF_REVIEW_DATABASE = "src/main/java/Databases/ReviewDatabase.csv";
+
+
+    public void updateReview(Review review) {
```

Comment:
This looks good, but I think this method should be moved into the ReviewGateway

Reply:
Sure, do u want me to do that? Or you want to do that? I don't wanna mess up your ReviewGateway just in case lol

Reply:
Let's both do it so we hopefully do not have to resolve any conflicts at a later merge

Reply:
ok so should we wait until this pull request is merged?  then send a new pull request? does that work?

Reply:
Yeah I think that works